cluster/gce/manifests/etcd.manifest

```diff
@@ -16,7 +16,7 @@
 "hostNetwork": true,
 "containers":[
     {
-    "name": "etcd-container",
+    "name": "etcd-container{{ suffix }}",
```

Comment:
IIRC, our monitoring depends on this being constant across instances.

But let's wait for @ptabor to actually confirm it.

Reply:
@GFilipek 
We are parsing the etcd-type (suffix) from the pod-name.

I don't have strong opinion here:
  1. parsing 'suffix' from globally-unique pod name is a little bit ugly, getting it straight from container name would be easier. 
  2. having a consistent layout (naming) of 'container's' within a POD is benefit that e.g. makes scripting easier. 
 
@vinayakankugoyal: What's the driving motivation on your end?

Reply:
My driving motivation is that, when I do `crictl ps -a`. I see 2 containers named `etcd-container`. This change will make it so we will see `etcd-container` and `etcd-events- container`, just a minor improvement(IMHO) to be able to better distinguish them.

Reply:
I think we should consider making this change. I had to debug my clusters master today and it is confusing seeing 2 container that have the same name. This way we can disambiguate what each of these containers does.

Reply:
I see... I keep forgetting that containerd does not understand 'pods'.

@GFilipek: - double checking it will not cause trouble for metrics-collection

Reply:
Etcd metrics will be collected, but new container name will cause problems with some monitoring dashboards and alerting.
This will probably trigger alerts for unknown container running on master and etcd-events data will be unavailable on dashboards which use container name filtering. We should not make this change before monitoring is prepared for new container name.

Reply:
I assume we are not going to backport it to <1.22 (as it might be breaking for other customers as well), so it gives a lot of time to adopt.